packages/turf-along/test.ts

```diff
@@ -2,11 +2,11 @@ import path from "path";
 import test from "tape";
 import { loadJsonFileSync } from "load-json-file";
 import { Units, featureCollection } from "@turf/helpers";
-import { along } from "./index";
+import { along } from "./index.js";
 import { Feature, LineString } from "geojson";
 
 const line: Feature<LineString> = loadJsonFileSync(
-  path.join(__dirname, "test", "fixtures", "dc-line.geojson")
+  path.join("test", "fixtures", "dc-line.geojson")
```

Comment:
Removing `__dirname` in the tests means that you can only run these tests with the cwd at the root of the package. I'm not sure that's a practical concern that's worth dealing with the `import.meta` nonsense.

Reply:
Easy enough to remedy. Will restore these references, and add a import.meta based __dirname where required.

Reply:
Its probably only really useful in the packages/turf tests that look into other packages, just to be extra careful there

Reply:
Reintroduced all these __dirname entries too just to be on the safe side.